src/main/java/Duke/Exception/DukeException.java

```diff
@@ -0,0 +1,75 @@
+package Duke.Exception;
+
+import java.io.IOException;
+
+public class DukeException extends Exception{
+    public static final String LINE = "____________________________________________________________";
+
+    public static void emptyTodoException() {
```

Comment:
Notice that for the methods you have here, you are repeating the same few print statement, so you may want to extract them out and create a new method.